apps/desktop/src/app/sync/protobuf.ts

```diff
@@ -21,6 +21,21 @@ export const NucliaProtobufConverter = (buffer: Uint8Array) =>
     }
   });
 
+const BASE64_MARKER = ';base64,';
+export const convertDataURIToBinary = (dataURI: string) => {
+  const base64MarkerIndex = dataURI.indexOf(BASE64_MARKER);
+  const base64Index = base64MarkerIndex === -1 ? 0 : dataURI.indexOf(BASE64_MARKER) + BASE64_MARKER.length;
+  const base64 = dataURI.substring(base64Index);
+  const raw = window.atob(base64);
+  const rawLength = raw.length;
+  const array = new Uint8Array(new ArrayBuffer(rawLength));
+
+  for (let i = 0; i < rawLength; i++) {
+    array[i] = raw.charCodeAt(i);
+  }
+  return array;
+};
+
```

Comment:
I think it could be done with:
```ts
const array = Uint8Array.from(raw);
```

Reply:
Sadly no, it's not that simple
<img width="1208" alt="image" src="https://user-images.githubusercontent.com/3293937/177995245-741fdbd4-6e79-4a42-90e8-9cf741babe17.png">